src/frontend/src/components/home/SearchablesRow.tsx

```diff
@@ -0,0 +1,181 @@
+import React from "react";
+import Box from '@mui/material/Box';
+import InputBase from '@mui/material/InputBase';
+import SearchIcon from '@mui/icons-material/Search';
+import { styled, alpha } from '@mui/material/styles';
+import enviroment from "../../enviroment";
+import AutoAwesomeIcon from '@mui/icons-material/AutoAwesome';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import windowDimention from "../../customHooks/WindowDimension";
+import CustomDropdown from "../../utilities/CustomDropdown";
+import CustomizedButton from "../../utilities/CustomizedButton";
+const SearchablesRow = () => {
+    const { windowSize, type } = windowDimention();
+
+
+    const searchableInnerStyle: any = {
+        search: {
+            backgroundColor: 'white',
+            border: '1px solid #000370',
+            opacity: 0.8,
+            marginTop: '0.7%',
+            color: 'white',
+            width: '15%',
+            fontFamily: 'BarlowMedium'
```

Comment:
@krtonga This project uses Material UI, styles for fonts, colors, space units, should be variables from the theme:

https://mui.com/material-ui/customization/theming/

Percentages should be avoided on padding, margin and grid items, to prevent undesired behaviors. Flexbox should be used instead.